interfaces/final-object.interface.ts

```diff
@@ -56,6 +57,7 @@ export interface ImageElement {
   resBucket: number;             // the resolution category the video falls into (for faster sorting)
   resolution: ResolutionString;  // e.g. `720`, `1080`, `SD`, `HD`, etc
   selected?: boolean;            // for batch-tagging of videos
+  frameRateDisplay?: string;      // displayed as XXfps
```

Comment:
I don't think this field is needed - as we can just append (the text) "fps" in the _template_ (the `.html` file) 👍